pkg/agent/util/net_linux.go

```diff
@@ -194,3 +194,9 @@ func ListenLocalSocket(address string) (net.Listener, error) {
 func DialLocalSocket(address string) (net.Conn, error) {
 	return dialUnix(address)
 }
+
+// IsInfraContainer return if a container is infra container according to the network namespace path.
+// Always return true on Linux platform, because kubelet only call CNI request for infra container.
+func IsInfraContainer(netNS string) bool {
```

Comment:
I feel this one might be put in the same place as hostNetNsPath(). Maybe move to server_linux/windows.go.

Reply:
Yes, I think we can put it in  server_linux/windows.go after I move the infra contaier out of IPAM service.